index.html

```diff
@@ -10217,56 +10217,50 @@ <h3>Value</h3>
 				<dt id="valuetype_string">string</dt>
 				<dd>Unconstrained value type.</dd>
 				<dt id="valuetype_token">token</dt>
-				<dd>One of a limited set of allowed enumerated values. The default value is defined in each attribute's Values table, as specified in the <a href="#enumerated-attribute-values">Enumerated Attribute Values</a> section.</dd>
+				<dd>One of a limited set of allowed values. The default value is defined in each attribute's Values table, as specified in the <a href="#enumerated-attribute-values">Attribute Values</a> section.</dd>
 				<dt id="valuetype_token_list"><dfn>token list</dfn></dt>
 				<dd>A list of one or more tokens.</dd>
 			</dl>
 			<p>These are generic types for states and properties, but do not define specific representation. See <a href="#state_property_processing">State and Property Attribute Processing</a> for details on how these values are expressed and handled in host languages.</p>
 		</section>
 	</section>
-	<section id="enumerated-attribute-values">
-		<h2>Enumerated Attribute Values</h2>
-		<p>When the ARIA attribute definition includes a table enumerating the attribute's allowed <span>values</span>, 
-		that attribute is a nullable <a data-cite="html/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>. 
-		Each value in the table is a keyword for the attribute, mapping to a state of the same name. </p>
+	<section>
```

Comment:
I see why you moved the "Multi-Value" section down to an H3 sub-section, but should this higher-level section get a new IDREF? (No strong preference. Approving as-is.)